engines/director/detection_tables.h

```diff
@@ -8734,6 +8759,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	WINGAME1("noddyfair", "", "Noddy PC.exe", "c612aa43e7ef55aa4fd2e1085fb1ef7a", 2197822, 850),
 
+	WINGAME1_l("norgesjakten", "", "STARTW95.EXE", "13b9b8939b4d35841805f97b9b298216", 2293279, Common::NB_NOR, 850),
```

Comment:
The same thing; all exe files must have tail detection. Put `t:`, and rerun detection; it will provide an updated entry.

Also, the file name here is way too generic, please add a second file.

Reply:
This new entry for Norgesjakten turned out to be bogus. I must have slipped up somewhere. Entry removed.
However, Sverigejakten was facing the same issue with a generic `STARTW95.exe` file name, so I added a second file for that game.